doc/whats_new/v1.0.rst

```diff
@@ -185,6 +185,10 @@ Changelog
 - |Efficiency| Improved speed of :func:`metrics.confusion_matrix` when labels
   are integral.
   :pr:`9843` by :user:`Jon Crall <Erotemic>`.
+- |Fix| :func:`metrics.top_k_accuracy_score`, works now also, if not all classes
+  of a multiclass problem are specified in the `y_true` parameter, when all
+  classes are therefore being specified in the parameter `labels`.
+  :pr:`19721` by :user:`Joris Clement <flyingdutchman23>`
```

Comment:
```suggestion

```suggestion

- |Fix| :func:`metrics.top_k_accuracy_score` now supports multiclass problems
  where only two classes appear in `y_true` and all the classes are specified in
  `labels`. :pr:`19721` by :user:`Joris Clement <flyingdutchman23>`
```

Reply:
Yeah, that is better.

Reply:
Fixed a small typo in [this commit](https://github.com/scikit-learn/scikit-learn/pull/19721/commits/02732f67b2d888cf6aa3193e8457f381dfa83f5e).